src/pages/Missions.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import Table from 'react-bootstrap/Table';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchMissions,
+  bookedMission,
+  cancelledMission,
+} from '../redux/mission/missions';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
+
```

Comment:
- [ ] when i navigate to rocket page and come back to this page, all my reserved mission is changed to unreserved mission, I guess it is because you run fetchMission on every render, the fetchMission should be called on initial render, this way you keep your state preserved